Documentation/troubleshooting.rst

```diff
@@ -134,6 +134,75 @@ running agent. Debugging of an individual endpoint can be enabled by running
     CPU 00: MARK 0x7dc2b704 FROM 3978 DEBUG: Handling ICMPv6 type=135
     CPU 00: MARK 0x7dc2b704 FROM 3978 DEBUG: ICMPv6 neighbour soliciation for address b21a8c0:d68a0000
 
+debuginfo
+=========
+
+The tool ``cilium debuginfo`` can print some useful debugging information from
+the Cilium API. The output is in Markdown format so this can be used when
+reporting a bug on our `issue tracker`_. To get a dump run
+
+::
+
+    cilium debuginfo -f debuginfo.md
+
+.. Note::
+
+          Please check the dump file for sensitive informations and strip it
+          away before sharing it with us.
+
+cilium-bugtool
+==============
+
+The ``cilium-bugtool`` let's you capture possibly useful informations about
+your environment for debugging. The tool is meant to be used for debugging a
+singel Cilium agent node. It collects command outputs and files from several
```

Comment:
singel

Reply:
Not sure what you mean, but this is just a draft which will be revisited when I am done with the review and remaining items for dropping the RFC tags.

Reply:
sorry. I was unhelpfully pointing out the typo. I keep seeing review request emails from github and forget to check the wip tag.